src/main/java/liquibase/ext/databricks/change/alterViewProperties/AlterViewPropertiesChangeDatabricks.java

```diff
@@ -0,0 +1,43 @@
+package liquibase.ext.databricks.change.alterViewProperties;
+
+import liquibase.change.DatabaseChange;
+import liquibase.change.DatabaseChangeProperty;
+import liquibase.database.Database;
+import liquibase.ext.databricks.change.AbstractAlterPropertiesChangeDatabricks;
+import liquibase.ext.databricks.database.DatabricksDatabase;
+import liquibase.servicelocator.PrioritizedService;
+import liquibase.statement.SqlStatement;
+import lombok.Setter;
+
+@Setter
+@DatabaseChange(name = "alterViewProperties", description = "Alter View Properties", priority = PrioritizedService.PRIORITY_DATABASE + 500)
```

Comment:
+ 500 is too much.
Classes from our extension may interfere when clients work with other databases, as for some workflows `supports` method is not called.
`PrioritizedService.PRIORITY_DATABASE` should be enough.
 If there is some parent class that already has PRIORITY_DATABASE, we can add `+1`, but usually it's not needed

Reply:
Changed to `PRIORITY_DATABASE`